src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -172,6 +171,7 @@ private bool Vote(ApplicationEngine engine, UInt160 account, ECPoint voteTo)
                 if (!state_validator.Registered) return false;
                 state_validator.Votes += state_account.Balance;
             }
+            state_account.VoteTo = voteTo;
```

Comment:
Could you add a UT for ensure this case? @Tommo-L

Reply:
Okay, I'll add it.